climpred/constants.py

```diff
@@ -9,18 +9,14 @@
     'bias',
     'std_ratio',
     'bias_slope',
-    'crps',
-    'crpss',
-    'less',
```

Comment:
Is less considered "probabilistic"? Should it be moved in the docs metrics page to the probabilistic heading then?

Reply:
it is